### PR TITLE
solution

### PR DESCRIPTION
**Pull request by deelray**
[DEMO LINK](https://deelray.github.io/react_dynamic-list-of-goods/)

### CODE REVIEW

**Review by ira-cherpak — APPROVED** [review]
Great

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
-// import * as goodsAPI from './api/goods';
+class App extends React.Component {
+  state = {
+    goods: [],
+  };
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+  selectGoods = (neededGoods) => {
```
Comment (aionova) [inline src/App.js:12]:
```suggestion
  selectGoods = (fetchGoods) => {
```

File: src/GoodList.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const GoodList = ({ goods }) => (
+  <ul>
+    { goods.map(good => (
+      <li style={{ color: good.color }} key={good.id}>{good.name}</li>
```
Comment (aionova) [inline src/GoodList.js:7]:
```suggestion
      <li style={{ color: good.color }} key={good.id}>
        {good.name}
      </li>
```

**Review by aionova — APPROVED** [review]
Awesome!

**Pull request closed without merging**